lib/src/signed_video_h26x_auth.c

```diff
@@ -334,13 +332,12 @@ verify_hashes_with_hash_list(signed_video_t *self,
     found_item_after_sei = (item->prev == sei);
     // Check if this |is_first_nalu_in_gop|, but not used before.
     found_next_gop = (item->nalu->is_first_nalu_in_gop && !item->used_for_linked_hash);
+    last_used_item = item;
     // If this is a SEI, it is not part of the hash list and should not be verified.
     if (item->nalu->is_gop_sei) {
       last_used_item = item;
```

Comment:
As said in the previous review comment. You've already set this (now on line 335). You do not need to set it again.